projects/js-packages/publicize-components/src/hooks/use-post-meta/index.js

```diff
@@ -27,13 +27,16 @@ export function usePostMeta() {
 			getShareMessageMaxLength()
 		);
 
+		const postShares = meta.jetpack_social_post_shares || [];
+
 		return {
 			isPublicizeEnabled,
 			jetpackSocialOptions,
 			attachedMedia,
 			imageGeneratorSettings,
 			isPostAlreadyShared,
 			shareMessage,
+			postShares,
```

Comment:
This may not be reliable because it will not be updated on the frontend once the value changes on the backend.